src/lattice-C.cpp

```diff
@@ -21,6 +21,9 @@ ElError ElLLLCtrlDefault_s( ElLLLCtrl_s* ctrl )
     ctrl->variant = EL_LLL_NORMAL;
     ctrl->recursive = false;
     ctrl->cutoff = 10;
+    ctrl->precisionFudge = 2.0f;
+    ctrl->minColThresh = 0;
+    ctrl->unsafeSzReduct = false;
```

Comment:
I think the Python interface on top of the C interface will be broken with this changeset until the equivalent changes are propagated though.